syntax = "proto3";
package dfi;

message DFICreateRingOnBufferRequest {
	string name = 1;
}
